Meshery-Adapters/consul-meshery-adapter/step1.md

```diff
@@ -14,9 +14,9 @@ Meshery can be downloaded, installed, and launched with a single command:
 
 Meshery is now available at host:`9081`.
 
-Find the **Meshery UI** tab is configured to be listening at `http://localhost:9081`/
+Find the **Meshery Server** tab is configured to be listening at `http://localhost:9081`/
 
-- Click on the **Meshery UI** tab
+- Click on the **Meshery Server** tab
```

Comment:
This too. Please change back to Meshery UI.

Reply:
ok @asubedy